third_party/proton/csrc/lib/Profiler/RocTracer/RoctracerProfiler.cpp

```diff
@@ -135,7 +136,7 @@ void processActivity(RoctracerProfiler::CorrIdToExternIdMap &corrIdToExternId,
                      const roctracer_record_t *record, bool isAPI,
                      bool isGraph) {
   switch (record->kind) {
-  case 0x11F1: // Task - kernel enqueued by graph launch
+  case kHipVdiCommandTask:
```

Comment:
Replacing the magic number is a NFC right? It's just now we have a named variable in the HIP headers?

Reply:
NFC? The hip headers are included with Triton so there should not be any issues as this is a drop in replacement as the enum maps to the same number.

Reply:
NFC - no functional change.